src/pages/Details/Details.scss

```
@@ -0,0 +1,178 @@
+@import '../../styles/variables.scss';
+
+.menuDetails {
+  color: #3d3d3d;
+  .menuBody {
+    padding: 100px 300px;
+    font-size: 16px;
+
+    .menuHeader {
+      display: flex;
+      // justify-content: center;
+      margin-top: 150px;
+
+      .menuInfo {
+        transform: translate(50%);
+        .menuName {
+          padding-bottom: 20px;
+
+          border-bottom: 1px solid;
+          h2 {
+            padding-bottom: 10px;
+            font-weight: bold;
+            font-size: 24px;
+            color: #111;
+          }
+          p {
+            padding: 10px 0;
+            font-size: $headline;
+          }
```

```suggestion
      .menuInfo {
        transform: translate(50%);
        
        .menuName {
          padding-bottom: 20px;
          border-bottom: 1px solid;
          
          h2 {
            padding-bottom: 10px;
            font-weight: bold;
            font-size: 24px;
            color: #111;
          }
          
          p {
            padding: 10px 0;
            font-size: $headline;
          }
```
불필요한 공백은 없애주시고, 새로운 셀렉터가 시작될 때는 꼭 가독성을 위해 줄바꿈을 해주세요!

---

넵 줄바꿈 수정하고 다시 푸쉬했습니다~!